package.json

```diff
@@ -54,57 +56,49 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-observe-visibility": "^1.0.0",
     "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
```

Comment:
I had to downgrade vuelidate because Vite and the common components were playing nicely with each other. This fixes errors resulting from that.
![validation mixin error](https://github.com/bcgov/business-create-ui/assets/122301442/51e3148f-31af-4132-a75a-6f0c723105ce)

Reply:
Nice find.